app/assets/stylesheets/components/summary-document.scss

```diff
@@ -0,0 +1,75 @@
+//*-------------- mixins --------------------*/
+@mixin bullet($color, $radius, $top: 5px) {
```

Comment:
Yeah, I thought about this – it would involve changing the markup in `output-templates` though. So will run this through with content/design first and revisit I think.